manual/source/classes/contributed_definitions/transport-structure.rst

```diff
@@ -0,0 +1,26 @@
+.. _Transport-Structure:
+
+===================
+Transport Phenomena
+===================
+
+.. index::
+   IntroductionTransport
+   TransportAppDef
+
+
+.. _IntroductionTransport:
+
+Introduction
+##############
+
+
+.. _TransportAppDef:
+
+Application Definitions
+#######################
+
+Work on handshaking between EPICS-controlled experiments and NeXus resulted in one application definition for temperature dependent IV curve measurements.
```

Comment:
What sort of transport phenomena? Perhaps ion?

Reply:
Under this category we plan to host multiple transport phenomena. The actual AppDef is to support Temperature Dependent I-V (or JV for engineers) measurements. In this case, electrical charge is transported and the temperature dependent current response as a function of applied voltage is recorded.

Reply:
edited to convey what NXiv_temp is specifically and how it is an example for describing a group of phenomena